bin/runRollup.js

```diff
@@ -143,6 +143,16 @@ function bundle ( options ) {
 			return bundle.write( options );
 		}
 
+		if ( options.targets ) {
+			var result = null;
+
+			options.targets.forEach( function ( target ) {
+				result = bundle.write(target);
+			});
+
+			return result;
```

Comment:
Come to think of it I think we can safely remove es6-promise. Node 0.12 and above support it, and I couldn't give a crap about 0.10 or IE (it's easy to polyfill if someone really wants it). Let's just use native `Promise.all` and I'll raise a separate issue about removing es6-promise